### PR TITLE
Fiscal year

### PR DESCRIPTION
**Pull request by sangamsth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,41 +9,71 @@
...
     $ bundle
 
-Or install it yourself as:
-
-    $ gem install nepali_calendar
 
 ## Usage
 
-##### Rails 3/4
+##### Rails 6.1.0+
```
Comment (sarojkh) [inline README.md:25]: Change all 'sh to'  'ruby', such that the code formatting is ruby based(and easy to scan too, maybe?)

See the following as comparison - sh vs ruby, respectively.
Screen Shot 2022-06-20 at 13 23 35

File: lib/exceptions/exceptions.rb [hunk 2]
```diff
@@ -33,7 +31,7 @@ class CommunicationError < Error
     attr_reader :code
 
     # Public: fallback if there is no response code on the object
-    NOCODE = 000
+    NOCODE = 0o00
```
Comment (sarojkh) [inline lib/exceptions/exceptions.rb]: Any reason for this change?

File: lib/nepali_calendar/ad_calendar.rb [hunk 3]
```diff
@@ -1,117 +1,117 @@
...
+        wday_name: Date::DAYNAMES[wday] }
+        formatted_date = Date.parse(NepaliCalendar::Calendar.new('', date).to_s)
+        formatted_date.beginning_of_week
+      end
+      
+      def end_of_week
+        date = { year: year, month: month, day: day, wday: wday, month_name: Date::MONTHNAMES[month],
+          wday_name: Date::DAYNAMES[wday] }
+          formatted_date = Date.parse(NepaliCalendar::Calendar.new('', date).to_s)
+          formatted_date.end_of_week
```
Comment (sarojkh) [inline lib/nepali_calendar/ad_calendar.rb]: The indentations on multiple methods in this file looks very very off.

**Review by sarojkh — CHANGES_REQUESTED** [review]

File: lib/exceptions/exceptions.rb (hunk 2 above)
Comment (sangamsth) [reply lib/exceptions/exceptions.rb]: this could be a typo mistake

**Review by sarojkh — CHANGES_REQUESTED** [review]
Lets fix the following as well: 
Screen Shot 2022-06-20 at 14 30 44
Screen Shot 2022-06-20 at 14 30 21
Screen Shot 2022-06-20 at 14 30 10
Screen Shot 2022-06-20 at 14 29 54
Screen Shot 2022-06-20 at 14 29 46
Screen Shot 2022-06-20 at 14 29 40
Screen Shot 2022-06-20 at 14 29 23
Screen Shot 2022-06-20 at 14 29 14
Screen Shot 2022-06-20 at 14 29 06
Screen Shot 2022-06-20 at 14 28 57
Screen Shot 2022-06-20 at 14 28 50
Screen Shot 2022-06-20 at 14 28 44
Screen Shot 2022-06-20 at 14 27 55
Screen Shot 2022-06-20 at 14 27 48
Screen Shot 2022-06-20 at 14 27 42
Screen Shot 2022-06-20 at 14 27 36
Screen Shot 2022-06-20 at 14 27 26
Screen Shot 2022-06-20 at 14 27 17
Screen Shot 2022-06-20 at 14 27 10
Screen Shot 2022-06-20 at 14 26 59
Screen Shot 2022-06-20 at 14 26 47
Screen Shot 2022-06-20 at 14 26 37
Screen Shot 2022-06-20 at 14 26 10
Screen Shot 2022-06-20 at 14 26 04
Screen Shot 2022-06-20 at 14 25 56
Screen Shot 2022-06-20 at 14 25 47
Screen Shot 2022-06-20 at 14 25 28

File: lib/exceptions/exceptions.rb [hunk 4]
```diff
@@ -31,17 +33,14 @@ class CommunicationError < Error
...
```
Comment (sarojkh) [inline lib/exceptions/exceptions.rb]: This is not an octal number. Not every cop violation in rubocop is valid; this is one of those instances.

**Review by sarojkh — CHANGES_REQUESTED** [review]
Can no longer run specs. 
Screen Shot 2022-06-20 at 17 47 29

File: nepali_calendar.gemspec [hunk 5]
```diff
@@ -1,7 +1,9 @@
-$:.push File.expand_path("../lib", __FILE__)
+# frozen_string_literal: true
+
+$LOAD_PATH.unshift File.expand_path('../lib', __dir__)
```
Comment (sarojkh) [inline nepali_calendar.gemspec]: Lets revert this hunk

**Review by sarojkh — CHANGES_REQUESTED** [review]

**Pull request merged**